src/jit/compiler.h

```diff
@@ -3495,6 +3495,10 @@ class Compiler
 
     void fgInline();
 
+    void fgRemoveEmptyFinally();
+
+    void fgCloneFinally();
+
```

Comment:
Has the source code for the bodies of these two methods already been checked in?